dev_tools/generate_documentation.rst

```diff
@@ -0,0 +1,55 @@
+Generate the rendered documentation with Quarto
+===============================================
+
+This document is here to assist this project's maintainers in updating the `documentation <https://goodchemistryco.github.io/Tangelo-Examples/>`_ of this repository.
+Notebooks are rendered using `Quarto <https://quarto.org/>`_, and the source files are deployed via github-pages.
+The required stepa to render the documentation after significant changes, like the addition of a new notebook, are described below.
```

Comment:
We describe below how to re-generate this documentation after changes in the repository, such as the modification or addition of new notebooks.